js/tinymce/classes/dom/EventUtils.js

```diff
@@ -239,6 +238,21 @@ define("tinymce/dom/EventUtils", [], function() {
 		}
 
 		/**
+  	 * Checks if the DOM has completely loaded
+  	 *
+  	 * @method isDomLoaded
+  	 * @return {Boolean} True/false if the DOM is loaded or not
+		 */
+		self.isDomLoaded = function () {
+			if (document.readyState === "complete" || (document.readyState === "interactive" && document.body)) {
```

Comment:
Could this be changed to just `return (document.readyState === "complete" || (document.readyState === "interactive" && document.body))` instead of the `if/else` block?